lgc/patch/RegisterMetadataBuilder.cpp

```diff
@@ -907,14 +920,21 @@ void RegisterMetadataBuilder::buildPsRegisters() {
 // Builds register configuration for compute/task shader.
 void RegisterMetadataBuilder::buildCsRegisters(ShaderStage shaderStage) {
   assert(shaderStage == ShaderStageCompute || shaderStage == ShaderStageTask);
-  auto entryPoint = m_pipelineShaders->getEntryPoint(shaderStage);
```

Comment:
clang now gives me an unused private field warning on m_pipelineShaders. Could you remove the field please? (Unless you have a forthcoming change that starts using it again.)

Reply:
Thank you for pointing out. I will remove it.